docs/content/_index.md

```diff
@@ -1,21 +1,48 @@
 ---
 date: "2021-06-28T00:00:00+00:00"
-title: "Grizzly: Deploy and Observe as One"
+title: "Grizzly"
 ---
-***Building and shipping has become one and the same.***
+## A Command Line utility for managing Grafana Resources
+Software engineers know how to version and deploy their resources. Tools like Git or CI enable
+reliable workflows that track changes, with meaningful review processes giving confidence in the
+expected outcomes.
 
-“I test in production and so do you.” - [Charity Majors](https://twitter.com/mipsytipsy?lang=en)
+Now, with **Grizzly**, you can have all this with Grafana resources, dashboards, datasources and
+more.
 
-Grizzly empowers you to provision dashboards using the outputs of your existing as-code workflows.
-Whether you use Terraform or Pulumi, Waypoint or Dagger, Grizzly is the essential last-mile for
-declarative, as-code observability of your existing workflows.
+Grizzly can be a valuable component in a number of workflows around Grafana resources.
 
-***Code, deploy and observe, together.***
+## Edit/Publish
+![Edit workflow](images/workflow-edit.png)
```

Comment:
Why is there a `2. ...` in that image when it's the first one being presented?

Reply:
Bah. They were originally one image with three stories. I should commit the excalidraw files for these so they can be edited. I'll remove the numbers.